dev/preview/install-vm-ssh-keys.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+THIS_DIR="$(dirname "$0")"
+PRIVATE_KEY_PATH="$HOME/.ssh/vm_id_rsa"
+PUBLIC_KEY_PATH="$HOME/.ssh/vm_id_rsa.pub"
+
+function log {
+    echo "[$(date)] $*"
+}
+
+function has-dev-access {
+    kubectl --context=dev auth can-i get secrets > /dev/null 2>&1 || false
+}
+
+if ! has-dev-access; then
+    echo Setting up kubeconfig
```

Comment:
Use `log` instead for consistent formatting :)

Reply:
Done